fmrest/cloudserver.py

```diff
@@ -0,0 +1,132 @@
+"""Subclass of Server, specifically for connecting via the 'new' FileMaker Cloud"""
+from typing import Dict, List, Optional, Union
+import pycognito
```

Comment:
To make `pycognito` optional (see comment in `setup.py`) but still make the file load without import errors, I would suggest, we catch the import error here, and then raise when the module would actually be used (see below):

```suggestion
try:
    import pycognito
except ImportError:
    _has_pycognito = False
else:
    _has_pycognito = True
```